Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -673,13 +673,13 @@ def malop_processes_command():
             if not wanted_machine:
                 continue
 
-            output = {}
-            for info in PROCESS_INFO:
-                if item.get('type', '') == 'filterData':
-                    output[info['header']] = dict_safe_get(item, ['filterData', 'groupByValue'])
+        output = {}
+        for info in PROCESS_INFO:
+            if info.get('type', '') == 'filterData':
```

Comment:
Why did you change this from item to info?
Won't it change the behavior here?

Reply:
The previous version which worked was that:
`for i in range(len(PROCESS_INFO)):
            if PROCESS_INFO[i]['type'] == 'filterData':
                output[PROCESS_INFO[i]['header']] = elements[element]['filterData']['groupByValue']`

Therefore I think item was a mistake and in addition, _info_ had the _type_ key and _item_ did not.